data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,280 @@ event "remnant: pantica void sprites"
 event "remnant: cognizance timer"
 	system "Nenia"
 		fleet "Ember Waste Node" 6000
+
+
+
+mission "Remnant: Expanded Horizons Astral 1"
+	name "Prep for an expedition to Sagittarius A*"
+	description "To prepare for an expedition, you need to go to <destination> and have 6 space available for a new outfit."
+	blocked "You need at least 1 free bunk to take this mission."
+	passengers 1
+	minor
+	to offer
+		has "Remnant: Expanded Horizons Quarg 3"
+		random < 25
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		conversation
+			`"In the starport cafeteria you come across Dawn sitting at a table looking worried. She smiles when she sees you, and waves you to the seat opposite her. "Back from galivanting around the galaxy with more tales to share?" she signs.`
+			choice
+				`	"Yes, as a matter of fact..."`
+				`	"No, just hunting Korath..."`
+				`	"No, just testing some stuff..."`
+				`	"You look worried. Is something happening?"`
+					goto worried
+			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
+			choice
+				`	"You look worried. Is everything alright?"`
+					goto worried
+				`	"You look distracted. We can talk later."`
+					defer
```

Comment:
Personally, I'm the sort to go for the first response if someone looks concerned or preoccupied about something; but there are plenty of people who, upon seeing that someone is distracted and not paying attention, will just exit the situation. Could be they feel insulted that the person isn't paying them much attention, or it could be that they genuinely want to let the person focus on whatever has their attention. Either way, though, I think it is a valid option.

Reply:
Suggestion: Have it be a question on whether the person wants some time. So: "You look distracted. Do you want me to come back later?"

Reply:
This defer has now been removed.